functions/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as functions from "firebase-functions";
+import * as firebaseAdmin from "firebase-admin";
+import * as corsLib from "cors";
+import { getAddress } from "ethers/lib/utils";
+
+const admin = firebaseAdmin.initializeApp();
+
+const cors = corsLib({
+  origin: true,
+});
+
+export const createCustomToken = functions.https.onRequest(
```

Comment:
can we comment in the code what a "custom token" is for?

Reply:
commented